ModuleDescriptionParser/Testing/TestData/parameter-file-with-pointfile-slicer-issue2979.params

```diff
@@ -1,2 +1,2 @@
-Input Fiducial File = input.fcsv
-Output Fiducial File = output.fcsv
+Input Fiducial File = input.fcsv
+Output Fiducial File = output.fcsv
```

Comment:
Nitpick: Could you check that this is only file with trailing spaces ? If not, could you make a dedicated commit fixing them ?

Reply:
Done.  Removed end of line spaces in all files (except tclap directory).